draft-ietf-tls-tls13.md

```diff
@@ -1909,6 +1909,7 @@ processed and transmitted as specified by the current active session state.
            client_key_exchange_RESERVED(16),
            server_configuration(17),
            finished(20),
+           key_update(21),
```

Comment:
handshake_type 21 is already taken by [rfc4366](https://tools.ietf.org/html/rfc4366#section-3.3) for `Certificate URL` (first free seems to be 24)